language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/package.scala

```diff
@@ -3,43 +3,35 @@
 
 package com.daml.ledger
 
-import java.time.Clock
 import java.util.UUID
 
 import org.apache.pekko.actor.ActorSystem
 
-import scala.concurrent.ExecutionContext
 import com.daml.lf.data.Ref
-import com.daml.ledger.api.auth.{
-  AuthServiceStatic,
-  Authorizer,
+import com.digitalasset.canton.ledger.api.auth.{
   Claim,
   ClaimActAsParty,
   ClaimAdmin,
   ClaimPublic,
   ClaimReadAsParty,
   ClaimSet,
 }
-import com.daml.logging.LoggingContext
-import com.daml.platform.localstore.InMemoryUserManagementStore
 
 package object rxjava {
 
-  private[rxjava] def untestedEndpoint: Nothing =
-    throw new UnsupportedOperationException("Untested endpoint, implement if needed")
   private val pekkoSystem = ActorSystem("testActorSystem")
   sys.addShutdownHook(pekkoSystem.terminate(): Unit)
 
-  private[rxjava] val authorizer =
-    new Authorizer(
-      () => Clock.systemUTC().instant(),
-      "testLedgerId",
-      "testParticipantId",
-      new InMemoryUserManagementStore(),
-      ExecutionContext.parasitic,
-      userRightsCheckIntervalInSeconds = 1,
-      pekkoScheduler = pekkoSystem.scheduler,
-    )(LoggingContext.ForTesting)
+//  private[rxjava] val authorizer =
+//    new Authorizer(
+//      () => Clock.systemUTC().instant(),
+//      "testLedgerId",
+//      "testParticipantId",
+//      new InMemoryUserManagementStore(NamedLoggerFactory.root),
+//      ExecutionContext.parasitic,
+//      userRightsCheckIntervalInSeconds = 1,
+//      pekkoScheduler = pekkoSystem.scheduler,
+//    )(LoggingContext.ForTesting)
```

Comment:
I see you moved from this authorizer to one in `ledgerServices`
Are they the same? Can this be properly removed?